### DISCUSSION TITLE
Divide Foliate in 2 panels to open 2 epubs simultaneously

### DISCUSSION THREAD

**Discussion by sebex007** [johnfactotum/foliate#1342] [Ideas]
Well, like the title says, divide Foliate in 2 panels to open 2 epubs simultaneously. 
I would like, for example, to read the same version of an epub in both English and Spanish version simultaneously.

**Comment by johnfactotum** [comment]
You can open multiple files, or the same file, in multiple windows. Then arrange the windows side by side.

**Reply by sebex007** [reply]
Yes. You can do that. It's like opening multiple instances of Firefox windows, each with a different site.
I'm not sure if adding tabs to your program is difficult, but it will significantly improve with them!!
